cmd/libs/go2idl/args/args.go

```diff
@@ -71,11 +74,14 @@ type GeneratorArgs struct {
 }
 
 func (g *GeneratorArgs) AddFlags(fs *pflag.FlagSet) {
+	//FIXME: can we simplify operation to one input file, one output?
```

Comment:
Unfortunately no - we need to identify which other packages will be generated for protobuf so we know whether to embed the message definition into the particular package, or to reuse an existing definition.